SL/Controller/Part.pm

```diff
@@ -1208,6 +1227,101 @@ sub build_bin_select {
   );
 }
 
+sub save {
+  my ($self) = @_;
+  my @errors = ();
+
+  my $is_new = !$self->part->id;
+
+  # check that the part hasn't been modified
+  unless ( $is_new ) {
+    $self->check_part_not_modified or
+      return [t8('The document has been changed by another user. Please reopen it in another window and copy the changes to the new window')];
+  }
+
+  if (    $is_new
+       && $::form->{part}{partnumber}
+       && SL::DB::Manager::Part->find_by(partnumber => $::form->{part}{partnumber})
+     ) {
+    return $self->js->error(t8('The partnumber is already being used'))->render;
```

Comment:
Das sieht irgendwie falsch aus, da save ja eigentlich ein array ref zurück liefern soll. Funktioniert, da die benutzte partnumber schon vorher abgefangen wird (in check_form). War vermutlich schon vorher so,, finde ich aber nicht richtig.

Reply:
Habe ich entfernt, war überflüssig.